flink/sbt/sbt.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Using SBT to create a Flink job
+===============================
+
+Motivation
+----------
+
+Maven is a widely accepted build tool for projects that target the Java platform. The Radicalbit Fast Data Distribution provides an archetype to kickstart your development projects using Maven, but Scala developers may find more compelling to use the Maven alternative of choice for the Scala ecosystem, SBT (Simple Build Tool).
+
+Basic 'archetype'
+-----------------
+
+Currently, SBT does not provide built-in capabilities to scaffold a project like Maven archetypes do.
```

Comment:
I agree; furthermore, SBT 1.0 should introduce native scaffolding. For the time being, I would keep things simple and not require users to adopt plugins.